examples/graphql-client/graphql_client.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/graphql;
+import ballerina/io;
+
+// User defined data types to perform client side data-binding
+type ResponseWithErrors record {|
+    *graphql:GenericResponseWithErrors;
+    Data data;
+|};
+
+type Response record {|
+    *graphql:GenericResponse;
+    Data data;
+|};
+
+type Data record {|
+    Person profile;
+|};
+
+type Person record {
+    string name;
+    int age;
+};
+
+public function main() returns error? {
+    // Creates a new client with the backend URL.   
+    graphql:Client graphqlClient = check new ("http://localhost:4000/graphql");
+
+    string document = "{ profile { name, age } }";
+    // The `execute()` remote function of graphql:Client takes a GraphQL document 
+    // as the required argument and sends a request to the specified backend URL
+    // seeking a response. On the retrieval of a successful response, the client
+    // tries to perform data binding for the user-defined data type. On failure to
+    // retrieve a successful response or when the client fails to perform data
+    // binding, a graphql:ClientError will be returned.
+
+    ResponseWithErrors response = check graphqlClient->execute(document);
```

Comment:
Shall we add an error handling case here to make it clear?

Reply:
To make the bbe simple error handling is omitted here. 
Similar pattern followed in other client examples (http, udp, tcp etc)

Reply:
Agreed.